newsapi/utils.py

```diff
@@ -54,6 +54,13 @@ def validate_datetime_str(datetimestr):
 
 if PY3:
 
+    def is_valid_string_or_list(var):
```

Comment:
Made this a separate function because _some_ of the data type checks only need a single `str`, but some can take a `list` of `str`. I plan to make some more pull requests to better align with the API that will make use of these functions.